src/referencePackageSourceGenerator/GenerateProjects/PackageProjectTemplate.xml

```diff
@@ -3,6 +3,8 @@
 
   <PropertyGroup>
     <TargetFrameworks>#TargetFrameworks#</TargetFrameworks>
+	<LangVersion>latest</LangVersion>
+	<NoWarn>8632</NoWarn>
```

Comment:
This should include $(NoWarn).  Please moved to https://github.com/dotnet/source-build-reference-packages/blob/main/src/referencePackages/Directory.Build.props#L47 so that all warning suppressions are centrally located.  A comment should be added like the other NoWarns indicating the reason.

Without this the generated code has numerous warnings.

```suggestion
	<NoWarn>$(NoWarn);8632</NoWarn>
```